save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -19,16 +23,35 @@ import web.cssom.ClassName
 import web.file.File
 import web.http.FormData
 
-val cosvFileManagerComponent: FC<Props> = FC { props ->
+val cosvFileManagerComponent: FC<Props> = FC { _ ->
     useTooltip()
 
-    val (cosvAvailableFiles, setCosvAvailableFiles) = useState<List<String>>(emptyList())
+    @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
+    val organizationSelectForm = selectFormRequired<String>()
 
+    val (cosvAvailableFiles, setCosvAvailableFiles) = useState<List<String>>(emptyList())
     val (filesForUploading, setFilesForUploading) = useState<List<File>>(emptyList())
 
+    val (userOrganizations, setUserOrganizations) = useState(emptyList<OrganizationDto>())
+    val (selectOrganizations, setSelectOrganizations) = useState("")
+
+    useRequest {
+        val organizations = get(
```

Comment:
maybe select only organization which has flag `allowBulkUpload`?

Reply:
I agree. added it